components/Chips/tests/unit/ChipsTests.m

```diff
@@ -394,4 +411,40 @@ - (void)testChipsWithDeleteEnabled {
   XCTAssertEqual(controlViewCount, (NSUInteger)1);
 }
 
+- (void)testChipViewDynamicTypeBehavior {
+  // Given
+  MDCChipsTestsFakeChipView *chipView = [[MDCChipsTestsFakeChipView alloc] init];
+  UIFont *titleFont = [UIFont systemFontOfSize:14.0 weight:UIFontWeightMedium];
+  MDCFontScaler *fontScaler = [[MDCFontScaler alloc] initForMaterialTextStyle:MDCTextStyleBody2];
+  titleFont = [fontScaler scaledFontWithFont:titleFont];
+  titleFont = [titleFont mdc_scaledFontAtDefaultSize];
+  chipView.titleFont = titleFont;
+  chipView.mdc_adjustsFontForContentSizeCategory = YES;
+  chipView.mdc_legacyFontScaling = NO;
+  chipView.titleLabel.text = @"Chip";
+  [NSNotificationCenter.defaultCenter
+      postNotificationName:UIContentSizeCategoryDidChangeNotification
+                    object:nil];
+  CGFloat defaultFontSize = chipView.titleLabel.font.pointSize;
+
+  // When
+  if (@available(iOS 10.0, *)) {
```

Comment:
at the begining of the test add
if (!@available(iOS 10.0, *)) {
return;
}

we only want to test 10. so I would make the exit early from the test at the begining of the function. Its part of the given that we are only testing 10.

Reply:
umm. It seems `!@available(iOS 10.0)` is not a valid guard statement. I can move the whole test into `@available(iOS 10.0)`.

Reply:
Addressed in 865300029f6556aaf570620546e5bcb94e64d6f6